deploy/crds/jaegertracing.io_jaegers_crd.yaml

```diff
@@ -5651,6 +5651,9 @@ spec:
                   additionalProperties:
                     type: string
                   type: object
+                nodePort:
```

Comment:
Wasn't this caught during the CI process? We should fail the build if the generated files aren't synchronized.